dsl/seed/jenkinsfiles/Jenkinsfile.prod.prepare

```
@@ -0,0 +1,420 @@
+import org.jenkinsci.plugins.workflow.libs.Library
+import java.util.regex.Pattern
+
+@Library('jenkins-pipeline-shared-libraries')_
+
+// TODO: add parameters
+// 1. <prj>_RELEASE_BRANCH
+// 2. QUARKUS_VERSION
+// 3. 
+
+seedConfig = [:]
+mainBranchConfig = [:]
+repositoriesToBranch = []
+
+pipeline{
+    agent{
+        label 'kie-rhel8 && !built-in'
```

```suggestion
        label 'rhel8 && !built-in'
```

we don't need 16GB for yaml manipulation :D

---

fixed :smile: